index.html

```diff
@@ -295,26 +295,30 @@ <h6 class="title-tag-text">Terminal</h6>
                     so if you'd like to chat please get in touch.
                 </p>
             </div>
-            <form class="contact-form" action="contact">
-                <label for="name" class="fullname">
-                    <input class="name" id="name" type="text" placeholder="Full name">
-                </label>
-                <label for="email" class="email">
-                    <input class="email-address" id="email" type="text" placeholder="Email address">
-                </label>
-                <label for="feedback" class="feedback">
-                    <textarea class="textarea"  name="feedback" placeholder="Enter text here" id="feedback" cols="30" rows="10"></textarea>
-                </label>
+            <form class="contact-form" action="https://formspree.io/f/xqkjaerj" method="post">
+                <ul>
+                    <li class="fullname">
+                        <input class="name" id="name" type="text" placeholder="Full name" maxlength="30" required>
+                    </li>
+                    <li class="email">
+                            <input class="email-address" id="email" type="email" placeholder="Email address" required>
+                    </li>
+                    <li class="feedback">
+                        <textarea class="textarea"  name="feedback" placeholder="Enter text here" id="feedback" cols="30" rows="10" maxlength="500" required></textarea>
+                    </li>
+                </ul>
                <button type="submit" class="get-in-touch">Get in touch</button>
             </form>
-            <div class="social-icons bottom">
+            <div class="social-icons-bottom">
                 <a href="https://twitter.com/MilaAyalew"><i class="fa-brands fa-twitter"></i></a>
                 <a href="https://www.linkedin.com/in/miliyon-ayalew-210808131/"><i class="fa-brands fa-linkedin-in"></i></a>
                 <a href="https://github.com/Miliyonayalew/"><i class="fa-brands fa-github"></i></a>
                 <a href=""><i class="fa-brands fa-angellist"></i></a>                
             </div>
```

Comment:
- [Optional] Consider using an unordered list to list out your social icons. This is best practice for HTML semantics.😁